### PR TITLE
Feature/flag flags setup

### PR DESCRIPTION
**Pull request by ame-test**
Adds readme documentation to the application.
- How to setup admin roles.
- more info on the feature flag stuff.
- Improved `webpack` watch options for the container.

### CODE REVIEW

**Comment by ame-test** [comment]
Screenshot 2024-04-02 at 12 15 04

File: README.md [hunk 1]
```diff
@@ -20,6 +22,40 @@ docker compose up -d
 Visit http://host.docker.internal:3000/ to use the app.
 - http://localhost:3000 will not work.
 
+### Enabling all feature
+
+To try of the roles in the platform hub:
+- admin
+- limited_admin
+
+admin have access ot the feature flag page, which allows them to determine what features are avialable for use in the app.
```
Comment (jremes-ho) [inline README.md:31]: "ot" should be "to"?

**Review by jremes-ho — COMMENTED** [review]
Couple of comments.

**Review by jremes-ho — APPROVED** [review]
LGTM ✅

**Pull request merged**